lib/phli.js

```diff
@@ -71,18 +71,19 @@ Phli.prototype.getAddressHistory = function (address, callback) {
   });
 };
 
-Phli.prototype.getPermitInfo = function (permitID, callback) {
+Phli.prototype.getPermitInfo = function (permitID, options, callback) {
   var params = {
     $format: "json"
   };
+  for(option in options) {
```

Comment:
Looks like this global `option` is failing the TravisCI build -- would you mind fixing this?